server/e2e/api/specs/server-info.e2e-spec.ts

```diff
@@ -73,11 +73,11 @@ describe(`${ServerInfoController.name} (e2e)`, () => {
       const { status, body } = await request(server).get('/server-info/features');
       expect(status).toBe(200);
       expect(body).toEqual({
-        clipEncode: false,
-        configFile: true,
-        facialRecognition: false,
+        clipEncode: true,
+        configFile: false,
+        facialRecognition: true,
         map: true,
-        reverseGeocoding: false,
+        reverseGeocoding: true,
```

Comment:
Why does this need to be changed?

Reply:
Some of the e2e tests that are now in the jobs suite, would actually run all the jobs, so it was configured with a config file that disabled it. I've reverted those changes for the api suite, so it's back to the defaults again.

Reply:
Ahh, gotcha